### DISCUSSION TITLE
Question about using DeepH preprocessing

### DISCUSSION THREAD

**Discussion by nicherry** [mzjb/DeepH-pack#74] [Q&A]
Why as I try to preprocess the output file from ABACUS, it tells me that "EFERMI" can't be found in log file? Appreciate it if there is anyone can give a helping hand.

**Comment by mzjb** [comment]
The log file format in ABACUS frequently changes, and DeepH-pack does not support all versions of ABACUS. You can either switch to a supported version of ABACUS or modify the parsing script to accommodate the version you are using.
